setup.py

```diff
@@ -54,13 +75,15 @@ def get_versions():
              # former naming schema?
              "-mt"
            ]
-for suf in suffixes:
-    lib = find_library("boost_python%s" % suf)
-    if lib is not None:
-        libs.append("boost_python%s" % suf)
+    for suf in suffixes:
+        print("finding boost_python%s" % suf )
+        lib = find_library("boost_python%s" % suf)
+        print(lib)
+        if lib is not None:
+           libs.append("boost_python%s" % suf)
         break
```

Comment:
You forgot  a level of indent here for the break. That's what breaks the Mac build.